### PR TITLE
[3.0][docs] update example in README

### PR DESCRIPTION
**Pull request by Shawn-Hx**

### CODE REVIEW

**Comment by gtk96** [comment]
@Shawn-Hx I think we should keep two quick starts at the same time.  What do you think?

File: README.md [hunk 1]
```diff
@@ -25,7 +25,47 @@ This README is meant as a brief walkthrough on the core features of CDC Connecto
...
 
-## Usage for Table/SQL API
+## Quick Start
+
+### Usage for Pipeline Connectors
+
+The example shows how to continuously synchronize data, including snapshot data and incremental data, from multiple business tables in MySQL database to Doris for creating the ODS layer.
+
+1. Download and extract the flink-cdc-3.0.tar file to a local directory.
+2. Download the required CDC Pipeline Connector JAR from Maven and place it in the lib directory (automated pulling using a script can be optimized in the future).
```
Comment (ruanhang1993) [inline README.md:35]: Please delete `(automated pulling using a script can be optimized in the future)`

File: README.md [hunk 2]
```diff
@@ -25,7 +25,47 @@ This README is meant as a brief walkthrough on the core features of CDC Connecto
...
+3. Configure the FLINK_HOME environment variable to load the Flink cluster configuration from the flink-conf.yaml file located in the $FLINK_HOME/conf directory.
```
Comment (ruanhang1993) [inline README.md:36]: `export FLINK_HOME=/path/to/your/flink/home`

File: README.md [hunk 3]
```diff
@@ -25,7 +25,47 @@ This README is meant as a brief walkthrough on the core features of CDC Connecto
...
+
+pipeline:
+  name: mysql-sync-doris
+  parallelism: 4
+```
+5. Submit the job to Flink cluster.
+```bash
+# Submit Pipeline
+$ ./bin/flink-cdc.sh mysql-to-doris.yaml
+Pipeline "mysql-sync-kafka" is submitted with Job ID "DEADBEEF".
```
Comment (ruanhang1993) [inline README.md:61]: mysql-sync-doris ?

**Review by ruanhang1993 — APPROVED** [review]
LGTM.

**Pull request merged**